source/features/remove-checks-tab.tsx

```diff
@@ -0,0 +1,36 @@
+/*
+The `checks` tab in PR is removed unless you're owner
+*/
+
+import select from 'select-dom';
+import features from '../libs/features';
+import {safeElementReady} from '../libs/dom-utils';
+import {isOwnRepo} from '../libs/page-detect';
+
+async function init() {
+	if (isOwnRepo()) {
+		return;
+	}
+
+	await safeElementReady(`
+		[data-hotkey="g 3"]
+	`); // Wait for the tab to be loaded
+
+	const checksTab = select([
+		'[data-hotkey="g 3"]'
+	].join());
```

Comment:
`join` don’t make sense here, don’t copy-paste code without reading it 😅 

You can add `const checkTabs =` before `await` and skip the `select` altogether